utilities/backupable/backupable_db.cc

```diff
@@ -781,36 +782,25 @@ Status BackupEngineImpl::CreateNewBackupWithMetadata(
         manifest_fname.size(), 0 /* size_limit */, false /* shared_checksum */,
         progress_callback, manifest_fname.substr(1) + "\n");
   }
-
+  Log(options_.info_log, "begin add wal files for backup -- %lu",
+        live_wal_files.size());
```

Comment:
indent should match the open parenthesis (so "options_..." is vertically aligned with "live_wal_files...")